lua/bufferline/state.lua

```diff
@@ -574,19 +574,19 @@ end
 local function close_all_but_current()
   local current = nvim.get_current_buf()
   local buffers = m.buffers
-  for i, number in ipairs(buffers) do
+  for _, number in ipairs(buffers) do
     if number ~= current then
-      vim.fn['bufferline#bbye#delete']('bdelete', '', bufname(number))
+      require'bbye'.delete('bdelete', false, bufname(number))
```

Comment:
As we're now using lua here, we should add `local delete = requre'...'` at the start of the file, with the other imports.